src/odemis/gui/cont/tabs.py

```diff
@@ -3313,6 +3286,36 @@ def __init__(self, name, button, panel, main_frame, main_data):
         else:
             self.panel.pnl_fib_focus.Show(False)
 
+        # Manual focus mode initialization
+        # All the `blue` streams one for each detector to adjust for focus
```

Comment:
Seems like in this sentence a typo slipped in? Maybe a comma need to be add? Maybe the word All needs to be Add? What do you mean exactly?